src/screens/users/list/Users.tsx

```diff
@@ -120,11 +139,14 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
       // Refresh All
       const users = await this.getUsers(this.searchText, 0, skip + limit);
       const usersResult = users ? users.result : [];
+      const allUsers = await this.getUsers(undefined, 0, limit, true);
```

Comment:
Should not be sync, if you have a lot of records, it can be time consuming and will block the UI.
Whether we handle it like in Angular (async) or we remove it.

Reply:
couting records is that much time consuming ?

Reply:
Yes, the worst case is in the Logs, check in Angular :-).

Reply:
In this case we also have a problem with all the get items where when getting -1 we count with await